src/Servers/Kestrel/Core/src/Internal/Http/Http1ContentLengthMessageBody.cs

```diff
@@ -145,11 +145,13 @@ private void CreateReadResultFromConnectionReadResult()
         {
             if (_readResult.Buffer.Length > _inputLength)
```

Comment:
I think these two branches can be merged.

Reply:
Does slice noop if it doesn't actually need to slice?

Reply:
It's cheap enough that you shouldn't care.

Reply:
Is it?

Reply:
Yes